lib/game.dart

```diff
@@ -314,50 +313,45 @@ class SimpleGame extends BaseGame {
 /// Provided it with a [Game] instance for your game and the optional size of the widget.
 /// Creating this without a fixed size might mess up how other components are rendered with relation to this one in the tree.
 /// You can bind Gesture Recognizers immediately around this to add controls to your widgets, with easy coordinate conversions.
-class EmbeddedGameWidget extends StatefulWidget {
+class EmbeddedGameWidget extends LeafRenderObjectWidget{
   final Game game;
   final Position size;
-
   EmbeddedGameWidget(this.game, {this.size});
 
-  @override
-  State<StatefulWidget> createState() {
-    return _EmbeddedGameWidgetState();
-  }
-}
-
-class _EmbeddedGameWidgetState extends State<EmbeddedGameWidget> {
-  _EmbeddedGameWidgetState();
 
   @override
-  void initState() {
-    super.initState();
-    WidgetsBinding.instance.addPostFrameCallback(_afterLayout);
-  }
-
-  @override
-  void didUpdateWidget(EmbeddedGameWidget oldWidget) {
-    super.didUpdateWidget(oldWidget);
-    WidgetsBinding.instance.addPostFrameCallback(_afterLayout);
-  }
+  RenderBox createRenderObject(BuildContext context) {
+    if(size != null){
+      return RenderEmbeddedGameBox(
+          _GameRenderBox(context, game),
+          widgetSize: this.size
```

Comment:
wisgetSize: this.size,

Reply:
I don't think wisgetSize wont break anything

Reply:
I was talking about the comma, I misspelled the var name hehe